extensions/gopass/src/details.tsx

```diff
@@ -0,0 +1,58 @@
+import { Action, ActionPanel, Clipboard, closeMainWindow, Icon, List, showHUD, showToast, Toast } from "@raycast/api";
+import { useEffect, useState } from "react";
+import gopass from "./gopass";
+import { humanize } from "./utils";
+import { copyPassword, pastePassword } from "./index";
+
+async function copy(key: string, value: string): Promise<void> {
+  Clipboard.copy(value);
```

Comment:
I don't know if it is important but you're using `await` on `Clipboard.paste`.

Reply:
Yes, you are right. I just forgot about it.